src/middlewared/middlewared/plugins/pool.py

```diff
@@ -3049,7 +3051,12 @@ def export_keys(self, job, id):
             Bool('recursive', default=False),
             List(
                 'datasets', items=[
-                    Dict('dataset', Str('name', required=True), Str('passphrase', required=True, empty=False)),
+                    Dict(
+                        'dataset',
+                        Str('name', required=True, empty=False),
+                        Str('key', validators=[Range(min=64, max=64)]),
```

Comment:
Hm, why would one need to provide a key to unlock a dataset?

Reply:
Edge case, where user has imported a dataset perhaps which was exported from a different machine. The same can be used to replace the entire usage of `key_file` if desired